resources/type-system/subtyping.md

```diff
@@ -121,12 +134,31 @@ We say that a type `T0` is a subtype of a type `T1` (written `T0 <: T1`) when:
 
 - **Right Top**: `T1` is a top type (i.e. `Object`, `dynamic`, or `void`).
```

Comment:
It's being discussed [here](https://github.com/dart-lang/language/issues/141) whether `Object` should be a top type, or it should be spelled `Object?`. If the latter is chosen, 'Right Top' should only mention `dynamic` and `void`.